plugin/adsk/plugin/plugin.cpp

```diff
@@ -53,6 +53,13 @@
 #include <mayaUsd/ufe/Global.h>
 #endif
 
+#if defined(MAYAUSD_VERSION)
+#define STRINGIFY(x) #x
+#define TOSTRING(x) STRINGIFY(x)
+#else
+#error "MAYAUSD_VERSION is not defined"
+#endif
```

Comment:
Macro trick to stringify a define. I cause compiler error if the MAYAUSD_VERSION is not defined.